app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+angular.module("openshiftConsole")
+  /**
+   * Widget for entering PVC information
+   *
+   * model:
+   *   The model for the input. The model will either use or add the following keys:
+   *     {
+   *       name: "",
+   *       accessMode: "",
+   *       Capacity: ""
+   *     }
+   *
+   */
+  .directive("oscPersistentVolumeClaim", function(){
+    return {
+      require: '^form',
```

Comment:
I left the controller and link in there for *possible* future enhancements, but I will remove it for now.